### DISCUSSION TITLE
Is this project still being actively maintained?

### DISCUSSION THREAD

**Discussion by day-jeff** [thuliteio/doks#1032] [General]
It looks like PRs haven't been review or merged for multiple months, so I'm wondering if this project is still being maintained. A little worried about taking a dependency if it's becoming stale.

**Comment by h-enk** [comment]
Hi Jeff, I understand your concern. But know that I'm working hard (behind the scenes) on getting the new **Doks 1.0**  ready for shipment. Doks 1.0 is completely restructured and now fully based on the new [Hyas v2.x](https://github.com/gethyas/hyas), making it e.g. super easy to update and add plugins and themes.

I'm almost there, but I can't give you an exact date when it will be available. The new setup is implemented, and all (improved) functionalities are there.  The only thing left now is updating some styling and updating documentation. Know that it has my priority to ship Doks 1.0 ASAP.

To give you a sneak preview, here's e.g. what the new FlexSearch implementation looks like (and it's set up way smarter under the hood):

![Snag_101a9b44](https://user-images.githubusercontent.com/3902872/229280007-de0b461f-617f-4e24-b0e3-05b00cb86a29.png)
